packages/shared_preferences/shared_preferences_android/test/shared_preferences_android_test.dart

```diff
@@ -48,33 +49,30 @@ void main() {
         final Match? match = setterRegExp.matchAsPrefix(methodCall.method);
         if (match?.groupCount == 1) {
           final String valueType = match!.group(1)!;
-          final String key = (methodCall.arguments['key'] as String?)!;
-          final Object value = (methodCall.arguments['value'] as Object?)!;
+          final String key = methodCall.arguments['key'] as String;
+          final Object value = methodCall.arguments['value'] as Object;
```

Comment:
Don't these need `!`s? I would expect you to get an analyzer warning about using `as` to convert from nullable to non-nullable.

Reply:
I wondered the same thing, but as it gave me no warnings left it out.